sorc/ncep_post.fd/WRFPOST.f

```diff
@@ -199,20 +197,11 @@ PROGRAM WRFPOST
 !open namelist
        open(5,file='itag')
        read(5,nml=model_inputs,iostat=itag_ierr,err=888)
-       !print*,'itag_ierr=',itag_ierr
 888    if (itag_ierr /= 0) then
        print*,'Incorrect namelist variable(s) found in the itag file,stopping!'
        stop
        endif
        
-       if (me==0) print*,'fileName= ',fileName
-         if (me==0) print*,'IOFORM= ',IOFORM
-         !if (me==0) print*,'OUTFORM= ',grib
-         if (me==0) print*,'OUTFORM= ',grib
-         if (me==0) print*,'DateStr= ',DateStr
-         if (me==0) print*,'MODELNAME= ',MODELNAME
-         if (me==0) print*,'SUBMODELNAME= ',SUBMODELNAME
```

Comment:
I agree with this change, however, it is sometimes important to see what the namelist is being set in the Fortran code.
I would recommend replacing the highlighted section with:
```fortran
if (me == 0) write(6, model_inputs)
```
Similarly at line 282.
```fortran
if (me == 0) write(6, nampgb)
```

Reply:
Done in  [dccad7f](https://github.com/NOAA-EMC/UPP/pull/705/commits/dccad7fe92592fbb2814aa82e5e4edc6fd0cfef2)